megamek/src/megamek/client/ui/swing/boardview/StepSprite.java

```diff
@@ -465,38 +465,32 @@ private void drawMovementCost(MoveStep step, boolean isLastStep,
 
         // If the step is using a road bonus, mark it.
         if (step.isOnlyPavement() && e.isEligibleForPavementBonus()) {
-            costStringBuf.append("+");
+            costStringBuf.append('+');
         }
 
         // Show WiGE descent bonus
         for (int i = 0; i < step.getWiGEBonus(); i++) {
-            costStringBuf.append("+");
+            costStringBuf.append('+');
         }
 
         // If the step is dangerous, mark it.
         if (step.isDanger()) {
-            costStringBuf.append("*");
+            costStringBuf.append('*');
         }
 
         // If the step is past danger, mark that.
         if (step.isPastDanger()) {
-            costStringBuf.insert(0, "(");
-            costStringBuf.append(")");
+            costStringBuf.insert(0, '(');
+            costStringBuf.append(')');
```

Comment:
Why the swap from double to single quotes?

Reply:
Slightly better performance, which adds up the more swaps are done.